internal/mcheck/mod.go

```diff
@@ -47,49 +47,74 @@ func main() {
 	)
 }
 
-// run parses all the comments in ast.File
+// iterateOverLines iterates over the lines of a comment and
+// checks if the line is too long or contains a prefix
+// that should be ignored
+func iterateOverLines(pass *analysis.Pass, lines []string, c *ast.Comment) {
+	for j := 0; j < len(lines); j++ {
+		line := lines[j]
+		if checkPrefixes(line) {
+			continue
+		}
+		ifTooLong(line, pass, c)
+		if strings.HasPrefix(line, NoLint) {
+			// Skip next comment for block comment.
+			j++
+		}
+	}
+}
+
+// checkPrefixes checks if the line starts with any of the following
+// prefixes:
+//
+// - `//go:generate`
+// - `// http://`
+// - `// https://`
```

Comment:
I would treat that as [code comment](https://go.dev/doc/comment#code) (i.e. just indent the line)